tests/main/parallel-install-basic/task.yaml

```diff
@@ -0,0 +1,86 @@
+summary: Checks for parallel installation of sideloaded snaps
```

Comment:
I've been tyring to kill "sideload" as a term for a long time. These are just normal installs of a local snap file, with or without assertions.